submit.py

```diff
@@ -292,7 +290,8 @@ def write_general_header(self, f, mem=3000, walltime_hours=14,
         self.write_option(f, '--mem=%d'%(mem*1.1))
         if num_gpus:
             self.write_option(f, "--gres=gpu:%d"%num_gpus)
-        self.write_option(f, "--partition=%s" % self.config["parition"])
+        if "partition" in self.config['Cluster'].keys():
```

Comment:
Should probably be: 
`if "partition" in self.config['Cluster']:`

No need to use `.keys()`